### PR TITLE
Fix command format in at/ap/as and update product name

### LINKED ISSUE #77: [PE-D][Tester A] at command example input is wrong
I think you forgot to put the space in between "at" and "n/..." in the output box when the input of an "at" command is wrong.

Copying the example and pasting it into the input box results in an unknown command error

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: Greg-Tan/ped#2

### LINKED ISSUE #80: [PE-D][Tester A] at/as/ap command error message does not show that the group field is optional
From further testing of correct inputs, the g/GROUP field is indeed optional and hence should be shown in the parameters as an optional field rather than not mentioning it completely

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Greg-Tan/ped#3

### LINKED ISSUE #110: [PE-D][Tester C] Traces of Address Book in command outputs
In command outputs, traces of Address Book still exist. 

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: NatLeong/ped#10

### PR DESCRIPTION
**Pull request by bachletuankhai**
- Add optional parameter GROUP in at/ap/as: Fixes #80 
- Fix typo in `at` example: Fixes #77 
- Change product name in log file and error message to HackLink and contact list. This is fixing incorrect terms, as pointed out in the forum. (see nus-cs2103-AY2324S2/forum#775 and nus-cs2103-AY2324S2/forum#752): Fixes #110

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -25,25 +25,17 @@
...
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_CATEGORY + "CATEGORY "
-            + "[" + PREFIX_GROUP + "GROUP]\n"
-            + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_CATEGORY + "participant "
-            + PREFIX_GROUP + "3 ";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the contact list. \n"
```
Comment (dhlee03) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:28]: It looks better to use original format for readability

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -37,18 +37,13 @@ public class EditCommand extends Command implements ReversibleCommand {
...
             + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
-            + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_PHONE + "PHONE] "
-            + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_GROUP + "GROUP]\n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + "Parameters: INDEX (must be a positive integer) " + "[" + PREFIX_NAME + "NAME] " + "[" + PREFIX_PHONE
```
Comment (dhlee03) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:40]: same here. previous one has better readability.

Comment (bachletuankhai) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:40]: It was my auto formatter, but ok i'll fix it

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (bachletuankhai) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:28]: Updated, please review

**Review by dhlee03 — APPROVED** [review]
LGTM.

**Pull request merged**